.github/workflows/benchmark.yml

```diff
@@ -28,7 +20,6 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v4
```

Comment:
actually if we ran this job on pull_request it was testing the main branch due to how `pull_request_target` work, now it should be fine